### PR TITLE
Payout splits to other projects bypass protocol fees. 

### PR DESCRIPTION
**Pull request by mejango**
Payout splits that go to other projects bypass fees.

in other words, only funds leaving the ecosystem incur the fee.

### CODE REVIEW

File: test/jb_eth_payment_terminal/distribute_payouts_of.test.js [hunk 1]
```diff
@@ -373,7 +476,7 @@ describe('JBETHPaymentTerminal::distributePayoutsOf(...)', function () {
               split.lockedUntil,
               split.allocator,
             ],
-            /*payoutAmount*/ Math.floor((AMOUNT_MINUS_FEES * split.percent) / SPLITS_TOTAL_PERCENT),
+            /*payoutAmount*/ Math.floor((AMOUNT_DISTRIBUTED * split.percent) / SPLITS_TOTAL_PERCENT),
```
Comment (drgorillamd) [inline test/jb_eth_payment_terminal/distribute_payouts_of.test.js:479]: This is disturbing me

File: contracts/JBETHPaymentTerminal.sol [hunk 2]
```diff
@@ -302,24 +302,48 @@ contract JBETHPaymentTerminal is
...
+        _fundingCycle,
+        _distributedAmount,
+        _feeDiscount
+      );
+
+      // Leftover distribution amount is also eligible for a fee since the funds are going out of the ecosystem to _beneficiary.
+      _feeEligibleDistributionAmount += _leftoverDistributionAmount;
+
+      // Take the fee.
+      _feeAmount = fee == 0 || _feeEligibleDistributionAmount == 0 || _projectId == 1
```
Comment (drgorillamd) [inline contracts/JBETHPaymentTerminal.sol:330]: Shouldn't we reorg this for "probabilistic gas saving"?
I'd say the order: _projectId == 1 || _feeEligibleDistributionAmount == 0 || fee==0

Comment (mejango) [reply contracts/JBETHPaymentTerminal.sol:330]: dope

File: contracts/JBETHPaymentTerminal.sol [hunk 3]
```diff
@@ -635,12 +663,18 @@ contract JBETHPaymentTerminal is
         JBConstants.SPLITS_TOTAL_PERCENT
       );
 
+      // The payout amount substracting any incurred fees (the platform project doesn't pays fee to itself)
```
Comment (mejango) [inline contracts/JBETHPaymentTerminal.sol:666]: typo "doesn't pay"

**Review by dhyon — APPROVED** [review]
looks good!

**Pull request merged**